### ISSUE TITLE
Sound Effects Cut Off in Browsers

### ISSUE THREAD

**Issue by KupoGames** [ruffle-rs/ruffle#3382]
Some sound effects are cut off incorrectly when Ruffle is running inside a browser, it seems to be sounds that have been cropped within Flash.

I've isolated the problem in this .swf file.
This sound effect works correctly in Flashplayer and Desktop Ruffle. It cuts off in browser Ruffle on Newgrounds (tested in Firefox and Chrome). I've noticed this issue in other games too and it makes for an unpleasant audio experience.
[sfx cutoff.zip](https://github.com/ruffle-rs/ruffle/files/6021196/sfx.cutoff.zip)

**Comment by sombraguerrero** [comment]
I previously closed this issue, but this is what I was describing in #3131 
At the time, I only saw it happening in the Firefox ESR edition, but I can confirm what Kupo is saying that as of the 3/20/21 nightlies when last I tested, this is now being exhibited across mainstream Chrome and FF Dev edition as well. I also concur that the desktop application does not now, nor did it previously exhibit this behavior. We all know what Pacman should sound like. Attaching Pacman as an example.
[pacman.zip](https://github.com/ruffle-rs/ruffle/files/6177932/pacman.zip)
